src/pages/index.js

```diff
@@ -23,16 +34,19 @@ const profileDescriptionInput = document.querySelector(
 );
 const profileEditForm = document.querySelector("#edit-profile-form");
 
-// Elements: Add Card Modal
+// Elements: Add Card Modal//
 const addNewCardButton = document.querySelector(".profile__add-button");
 const cardForm = document.querySelector("#add-card-form");
```

Comment:
**Could be improved**

We can get any form using their **unique** attribute `name` or `id` from `document.forms`

```js

const profileForm = document.forms["profile-form"];
const cardForm = document.forms["card-form"];

```

You need to add `name` or `id` first in `index.html` to achieve that.

More info here https://developer.mozilla.org/en-US/docs/Web/API/Document/forms